21-merge-lists.js

```diff
@@ -0,0 +1,23 @@
+var mergeTwoLists = function(l1, l2) {
+  const fakeHead = new ListNode(null)
+  let current = fakeHead
+
+  while (l1 && l2) {
+    if (l1.val <= l2.val) {
+      current.next = l1
+      current = current.next
+      l1 = l1.next
+    }
+
+    if (l1 && l2.val <= l1.val) {
```

Comment:
I'd use `else` here, so there's no need to check `l1` once again.

Reply:
Yes, you're right. My thinking was "we will use less iterations" but it's exactly the same to make an iteration over again or get into another conditional. Will do.